wf-core-tck-runner/jsonp-tck/pom.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright \(c\) "2022" Red Hat and others
+  ~
+  ~ This program and the accompanying materials are made available under the Apache Software License 2.0 which is available at:
+  ~  https://www.apache.org/licenses/LICENSE-2.0.
+  ~
+  ~  SPDX-License-Identifier: Apache-2.0
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <!-- Just for staging proflie access (-Pstaging) for unreleased Jakarta APIs -->
+  <parent>
+    <groupId>org.wildfly.tck</groupId>
+    <artifactId>core-tck-runner-parent</artifactId>
+    <version>1.0.0</version>
+  </parent>
+
+  <artifactId>jsonp-tck-runner</artifactId>
+  <name>WildFly Jakarta JSON-P TCK Runner</name>
+
+  <properties>
+    <excluded.groups>se</excluded.groups>
+  </properties>
+  <dependencies>
+
+    <dependency>
+      <groupId>jakarta.json</groupId>
+      <artifactId>jakarta.json-tck-tests</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>jakarta.ee.tck.coreprofile</groupId>
+      <artifactId>core-tck-jsonp-extension</artifactId>
+      <version>10.0.0-SNAPSHOT</version>
```

Comment:
LATER: This is odd

Reply:
updated jsonp-tck/pom.xml to not reference the `10.0.0-SNAPSHOT` as this does look wrong (switched to ${core.profile.tck.version}.